tests/routes/test_sessions.py

```
@@ -748,6 +748,51 @@ def test_get_session_context_with_all_params(
     assert "summary" in data
 
 
+def test_get_session_summaries(client: TestClient, sample_data: tuple[Workspace, Peer]):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_get_session_summaries`

Add return type annotation: `None`

(ANN201)